dev/AccessControl/SecurityDescriptorHelpers.h

```diff
@@ -0,0 +1,43 @@
+﻿#pragma once
+
+#include <Windows.h>
+#include <wil/resource.h>
+#include <wil/result.h>
+#include <processthreadsapi.h>
+
+
+namespace Security::Descriptors
+{
+    wil::unique_any_psid GetCurrentProcessSid()
+    {
+        wil::unique_any_psid result;
+        wil::unique_process_heap_ptr<TOKEN_GROUPS> tg;
```

Comment:
Looks like `TOKEN_GROUPS` is missing from https://github.com/microsoft/wil/blob/f9284c19c9873664978b873b8858d7dfacc6af1e/include/wil/token_helpers.h#L39 - consider adding it there and using it when a new wil update is available.  But there is `TOKEN_GROUPS_AND_PRIVILEGES`, maybe that helps?

Reply:
The helper for `TOKEN_GROUPS_AND_PRIVELEGES` doesn't work in this case. But I'll add a helper to WIL.